app/scripts/controllers/rif/lumino/index.js

```diff
@@ -29,17 +29,24 @@ export class LuminoManager extends AbstractManager {
       keyringController: this.keyringController,
     });
     this.luminoExplorer = new LuminoExplorer({
+      address: this.address,
       configurationProvider: this.configurationProvider,
     });
   }
 
   async initializeLumino (reconfigure = false) {
     const configuration = this.configurationProvider.getConfigurationObject();
     if (this.unlocked && isRskNetwork(this.network.id)) {
+      let hubEndpoint;
+      try {
+        hubEndpoint = await this.luminoExplorer.getHubConnectionUrl();
+      } catch (error) {
+        throw Error(error);
```

Comment:
what happens if there are no available hubs?

Reply:
If the explorer doesn't have any hub endpoints, the explorer will throw a 409 error that it will be an error here catched by the try catch, nifty will not be initialized and it will show an error on the ui. If for some reason the explorer give us an empty url or something like that where the try catch can't get the error the lumino sdk will throw us the error at the end saying in the ui that we can't connect to the hub properly.